api/proto/teleport/legacy/types/types.proto

```diff
@@ -2104,7 +2105,9 @@ message AuthPreferenceSpecV2 {
   // 1 is "legacy", 2 is "balanced-v1", 3 is "fips-v1", 4 is "hsm-v1".
   SignatureAlgorithmSuite signature_algorithm_suite = 20;
 
-  // SecondFactors is a list of supported second factor types.
+  // SecondFactors is a list of supported multi-factor types.
+  // 1 is "otp", 2 is "webauthn", 3 is "sso",
```

Comment:
In some places the `sso` factor is missing.

Reply:
I don't see any places where it's incorrectly missing, can you elaborate?